src/commands/gate/renderer.ts

```diff
@@ -0,0 +1,87 @@
+import chalk from 'chalk'
+
+import {SpanTags} from '../../helpers/interfaces'
+
+import {EvaluationResponse, RuleEvaluation} from './interfaces'
+
+const ICONS = {
+  FAILED: '❌',
```

Comment:
nitpick: I see other usages of icons in this repo as
```suggestion
  FAILED: chalk.bold.red('❌'),
```
do we need that as well?

Reply:
I don't think we need it, I copied this from the sarif/junitxml command